modules/prompt/functions/async

```diff
@@ -1 +1 @@
-../external/pure/async.zsh
\ No newline at end of file
+../external/async/async.zsh
```

Comment:
@wadkar, so did you have to manually recreate the symlink? It is odd if that's the case. Anyway, great that it got all sorted out.